src/vs/workbench/contrib/quickopen/browser/quickopen.contribution.ts

```diff
@@ -18,6 +18,10 @@ import { VIEW_PICKER_PREFIX, OpenViewPickerAction, QuickOpenViewPickerAction, Vi
 import { inQuickOpenContext, getQuickNavigateHandler } from 'vs/workbench/browser/parts/quickopen/quickopen';
 import { ContextKeyExpr } from 'vs/platform/contextkey/common/contextkey';
 import { KeybindingsRegistry, KeybindingWeight } from 'vs/platform/keybinding/common/keybindingsRegistry';
+import { registerQuickOpenConfiguration } from 'vs/workbench/contrib/quickopen/common/quickOpenConfig';
+
+// register quick open settings
+registerQuickOpenConfiguration();
```

Comment:
Changes to this file won't be needed after the setting is moved to task.contribution.ts.